Packs/Akamai_WAF/Integrations/Akamai_WAF/Akamai_WAF.py

```diff
@@ -4705,6 +4993,268 @@ def get_papi_property_rule_command(client: Client,
         return human_readable, {}, {}
 
 
+# Created by D.S. 2022-11-25
```

Comment:
Please remove comment

Reply:
We have two individuals working on the same integration and only using the Xsoar's Web based development interface.
We are planning to get the CI/CD pipeline setup within this year. However, before that is ready, we are probably going to keep track of some of our work in this manner.